db/seeds.rb

```diff
@@ -0,0 +1,60 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
+#
+# Examples:
+#
+#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
+#   Mayor.create(name: 'Emanuel', city: cities.first)
+
+require "csv"
+
+# MARKETS
+markets = CSV.read("seed_csvs/markets.csv", {headers: true, return_headers: true})
```

Comment:
Are you sure `headers: true` is what you want here? It doesn't look like any of the CSVs have a headers row on them. Try it without either `headers` or `return_headers` and see if it works.